roles/openshift_master/tasks/main.yml

```diff
@@ -1,7 +1,14 @@
 ---
 # tasks file for openshift_master
-- name: Install Origin
-  yum: pkg=openshift-master state=installed
+
+- name: Install docker
+  yum: pkg=docker state=installed
```

Comment:
The docker package should be coming from the docker role currently. Also, docker isn't needed unless the host is also a node as well, in which case, the openshift-node role should be applied to the host.